lib/vsc/install/shared_setup.py

```diff
@@ -275,12 +277,10 @@ def _fvs(msg=None):
 
 def _read(source, read_lines=False):
     """read a file, either in full or as a list (read_lines=True)"""
-    with open(source) as file_handle:
-        if read_lines:
-            txt = file_handle.readlines()
-        else:
-            txt = file_handle.read()
-    return txt
+    text = Path(source).read_text(encoding='utf8')
+    if read_lines:
+        return text.split()
```

Comment:
split on what? make it explicit it's on newlines

Reply:
you are right! but I forgot that python has splitlines(), I'll use that.